common/views/components/PageWrapper/PageWrapper.js

```diff
@@ -199,21 +199,23 @@ const PageWrapper = (Comp: NextComponent) => {
       }
 
       return (
-        <DefaultPageLayout
-          title={title}
-          description={description}
-          type={type}
-          canonicalUrl={canonicalUrl}
-          imageUrl={imageUrl}
-          pageJsonLd={pageJsonLd}
-          siteSection={siteSection}
-          analyticsCategory={analyticsCategory}
-          openingTimes={openingTimes}
-          globalAlert={globalAlert}
-          oEmbedUrl={oEmbedUrl}
-          pageState={pageStateWithToggles}>
-          <Comp {...props} />
-        </DefaultPageLayout>
+        <div>
```

Comment:
Why the div?

Reply:
accidentally left in after logging. Thanks. Removed.